scripts/generate_prefrosh.py

```diff
@@ -0,0 +1,18 @@
+import sqlalchemy
+import os
+from ruddock import config
+
+engine = sqlalchemy.create_engine(config.DEV.db_uri, convert_unicode=True)
```

Comment:
You should make this use a command line flag to set which environment to use. See the `run_server.py` script for an example of how to do that.